code/modules/reagents/reagent_containers/food/snacks/grown.dm

```diff
@@ -671,6 +671,22 @@
 	filling_color = "#66CEED"
 	plantname = "icechili"
 
+/obj/item/weapon/reagent_containers/food/snacks/grown/ghostpepper
+	name = "ghost pepper"
+	desc = "This pepper is hainted. And pretty spicy, too."
```

Comment:
It's usually associated with rural/uneducated people. I just think it sounds hilarious.